controllers/user.js

```diff
@@ -1 +1,319 @@
-// user authentication controller
\ No newline at end of file
+// user authentication controller
+import User from "../models/userSchema.js";
+
+const login = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+    if (!email || !password) {
+      return res.status(404).json({
+        message: "Please Fill all Fields",
+        success: false,
+      });
+    }
+
+    const user = await User.findOne({
```

Comment:
Create Model functions and get a response from it.
e.g getUser(params), addUser(params), updateUser(params) and reuse it controller